package.json

```diff
@@ -43,26 +43,26 @@
     "d3-fetch": "^1.2.0",
     "d3-interpolate": "^1.4.0",
     "d3-interpolate-path": "^2.2.1",
-    "d3-scale": "^3.2.1",
+    "d3-scale": "^3.2.3",
```

Comment:
I wonder whether this minor update will have any effect on the flowchart? Might be worth a small regression test just to make sure.

Reply:
This PR doesn't introduce any new functionality, so if we felt it necessary to write some new tests for the flowchart component, shouldn't we do that in a different PR, as needing more tests is a property of that component, not of this PR? I would suggest creating a new JIRA issue for this issue but we've already closed two on this very subject.

Secondly, you and I looked into testing the D3 functionality in this component about a week ago, and we both agreed that it was difficult enough that we felt it wasn't worthwhile - hence the closed JIRA issues.

Thirdly, Liam is reportedly working on some new tests in this area.

Reply:
Yeah I think if manual testing and the existing unit tests show no issues in this PR then it's fine, especially considering it's a patch update of the dependency. On the point of additional tests around this though, there's a bunch coming with the zoom refactor which should help double check this.

Reply:
Thanks both - for regression testing I meant 'manual QA' - sorry for the confusion!